examples/mqtt-paho/Makefile

```diff
@@ -0,0 +1,40 @@
+APPLICATION = mqtt-paho-example
+
+# If no BOARD is found in the environment, use this default:
+BOARD ?= native
+
+# This has to be the absolute path to the RIOT base directory:
+RIOTBASE ?= $(CURDIR)/../..
+
+# Comment this out to disable code in RIOT that does safety checking
+# which is not needed in a production environment but helps in the
+# development process:
+DEVELHELP ?= 1
+
+# Change this to 0 show compiler invocation lines by default:
+QUIET ?= 1
+
+#define to use MQTT Paho as a task using MQTTStartTask()
+CFLAGS += -DMQTT_TASK=1
```

Comment:
Useless if this is done in pkg/mqtt-paho/Makefile.include, so should be removed.

Reply:
Idem above